lib/vintage_net_wifi/wpa_supplicant.ex

```diff
@@ -49,6 +49,14 @@ defmodule VintageNetWiFi.WPASupplicant do
     GenServer.call(via_name(ifname), :signal_poll)
   end
 
+  @doc """
+	Get ready to recieve credentials via WPS
```

Comment:
```suggestion
    Enable reception of WiFi credentials via WPS
```

Do you know if this times out or does it have to be disabled?

Reply:
It times out after 10 minutes or so.